src/components/CollectionsAndPalettes/common/CollectionsToolbar/CollectionsToolbar.module.scss

```diff
@@ -135,6 +164,19 @@ a {
 .collaborators {
   display: flex;
   align-items: center;
+  @media (max-width: 520px) {
+    display: none;
+  }
+}
+
+.floatingShare {
+  display: none;
+  position: absolute;
+  right: 6%;
+  top: 7%;
```

Comment:
In order for this to behave more predictably, I think we should add a `position: relative` to the `pageContent` class in `components/CollectionsAndPalettes/CollectionsAndPalettes.module.scss` and use a fixed `top: 10px` in here.

Elements with `position:absolute` are being positioned relative to the closest parent that has a position other than static(default). So in this case, by making the page container have `position:relative`, the top / bottom / left / right on this element would position it relative to the page container, and not the entire page. So in that case we're only telling it to go 10px lower than where the header ends.